src/models/discussions.js

```diff
@@ -22,13 +21,18 @@ const discussion = new Schema({
         require: true
     },
     theme: {
+        type: String,
+        default: "info",
+        enum: ["info", "doubt", "rule"],
+        require: true
+    },
+    numberComments:{
         type: Number,
-        min: 1,
-        max: 3,
+        default: 0,
         require: true
     },
-    idAccount: [{ type: Schema.Types.ObjectId, ref: 'Accounts' }]
+    idAccount: [{ type: Schema.Types.ObjectId, ref: 'Accounts'}]
 });
 
-discussion.plugin(require('mongoose-autopopulate'));
-module.exports = mongoose.model('Discussions',discussion);
\ No newline at end of file
+discussions.plugin(require('mongoose-autopopulate'));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
discussions.plugin(require("mongoose-autopopulate"));
```